modules/configure-acs-policy-notifications.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-email.adoc
+[id="configure-acs-policy-notifications"]
+= Configuring policy notifications
```

Comment:
@tmalove We already have a module for this https://raw.githubusercontent.com/openshift/openshift-docs/rhacs-docs/modules/configure-policy-notifications.adoc

You can use that module as follows:
```
:toolname: Email
include::modules/configure-policy-notifications.adoc[leveloffset=+1]
```